content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -588,7 +592,7 @@ stage('Test') {
 <1> The `stash` step allows capturing files matching an inclusion pattern
 (`**/target/*.jar`) for reuse within the _same_ Pipeline. Once the Pipeline has
 completed its execution, stashed files are deleted from the Jenkins master.
-<2> The optional parameter to `agent`/`node` allows for any valid Jenkins label
+<2> The parameter in `agent`/`node` allows for any valid Jenkins label
```

Comment:
Yeah, I was trying to make a minimal change that covers both Scripted and Declarative.

The `label` parameter is now "in" the `agent` block, and arguably "in" the parentheses of `node('…')`.  I think it's reasonable.